types/p-event/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for p-event 1.3
+// Project: https://github.com/sindresorhus/p-event#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+export = pEvent;
+
+declare function pEvent<T, TRest = T>(emitter: pEvent.Emitter<T, TRest>, event: string | symbol, options: MultiArgsOptions): Promise<Array<T | TRest>>;
```

Comment:
event  should be of type string according to this: https://www.npmjs.com/package/p-event#event